apps/frontend/src/libs/components/create-poi-modal/create-poi-modal.tsx

```diff
@@ -1,54 +1,54 @@
+import React from "react";
+import { type FieldErrors } from "react-hook-form";
+
 import {
 	Button,
 	Input,
+	MapLocationField,
+	MapProvider,
 	Modal,
 	TextArea,
 } from "~/libs/components/components.js";
 import { useAppForm } from "~/libs/hooks/hooks.js";
+import { toastNotifier } from "~/libs/modules/toast-notifier/toast-notifier.js";
 import {
 	pointOfInterestCreateValidationSchema,
 	type PointsOfInterestRequestDto,
 } from "~/modules/points-of-interest/points-of-interest.js";
 
+import { DEFAULT_CREATE_POI_PAYLOAD } from "./libs/constants/constants.js";
 import styles from "./styles.module.css";
 
 type Properties = {
-	defaultLatitude: number;
-	defaultLongitude: number;
 	isOpen: boolean;
 	onClose: () => void;
-	onSubmit: (values: PointsOfInterestRequestDto) => void;
+	onSubmit: (payload: PointsOfInterestRequestDto) => void;
 };
 
 const CreatePOIModal = ({
-	defaultLatitude,
-	defaultLongitude,
 	isOpen,
 	onClose,
 	onSubmit,
 }: Properties): React.JSX.Element => {
-	const { control, errors, handleSubmit } =
+	const { control, errors, handleReset, handleSubmit } =
 		useAppForm<PointsOfInterestRequestDto>({
-			defaultValues: {
-				description: "",
-				location: {
-					coordinates: [defaultLongitude, defaultLatitude],
-					type: "Point",
-				},
-				name: "",
-			},
+			defaultValues: DEFAULT_CREATE_POI_PAYLOAD,
 			validationSchema: pointOfInterestCreateValidationSchema,
 		});
 
-	const handleFormSubmit = (values: PointsOfInterestRequestDto): void => {
-		onSubmit({
-			...values,
-			description: values.description || null,
-			location: {
-				...values.location,
-				coordinates: values.location.coordinates,
-			},
-		});
+	const handleFormSubmit = (payload: PointsOfInterestRequestDto): void => {
+		onSubmit(payload);
+		handleReset(DEFAULT_CREATE_POI_PAYLOAD);
+	};
+
+	const handleFormError = (
```

Comment:
```suggestion
	const handleFormError = (...
```
I know that we can't use toast, i need some hint how to do that

Reply:
in create action, call showError if api doesnt return pointOfInterest
if(!pointOfInterest){
  toastNotifier.showError("Text")
}
example:
https://github.com/BinaryStudioAcademy/bsa-2025-smartscapes/commit/b1649ccd4de593392c5921762205e2c4b2ea4939